client/client.go

```diff
@@ -269,6 +269,9 @@ type (
 	// QueryWorkflowWithOptionsResponse defines the response to QueryWorkflowWithOptions.
 	QueryWorkflowWithOptionsResponse = internal.QueryWorkflowWithOptionsResponse
 
+	// DescribeWorkflowExecution defines the response to DescribeWorkflow.
+	DescribeWorkflowExecution = internal.WorkflowExecutionDescription
```

Comment:
Though you don't _technically_ have to, I think you should also export the embedded field type

Reply:
What is the use case for the exported embedded field type?

Reply:
For tests, people often appreciate the ability to easily instantiate the objects our clients can return. In this case, you have to instantiate and then set one field at a time. I think it's a reasonable rule that "every struct that is exported should also have its exported fields' types exported" (unless there's a good reason to have a non-exported interface, which is possible but rare)